### ISSUE TITLE
THIS IS NOT UNSAFE AND WILL GET YOU BANNED

### ISSUE THREAD

**Issue by Belook12** [Glixerz/BlooketTokens#11]
This is unsafe and will get you banned immediatly! This is the wrong script

get the new scripts from https://schoolcheats.net/blooket

**Comment by kgsensei** [comment]
Don't worry, this is most likely a bot making false claims. The truth is, Blooket has no decent anti-cheat, and you will probably never get banned for using cheats. This individual has went around targeting Blooket cheat creators to try and hurt their work. Almost all of the claims made are false and can't be backed up.

**Comment by RoronoaDLuffy** [comment]
No, its just not up to date. that's why its not working because blooket owners update the anti cheat system everyday

**Comment by kgsensei** [comment]
Blooket has anticheat now?

**Comment by RoronoaDLuffy** [comment]
it always had anticheat

**Comment by kgsensei** [comment]
Never knew. It must have been really bad in the older times.
